types/chromecast-caf-receiver/cast.framework.breaks.d.ts

```diff
@@ -0,0 +1,98 @@
+import { Break, BreakClip } from "./cast.framework.messages";
+
+export = cast.framework.breaks;
+
+declare namespace cast.framework.breaks {
```

Comment:
Why do you need the nested namespaces here?

Reply:
Hi Daniel, thanks for reviewing.

I want these namespaces to exactly mirror the Chromecast CAF framework namespaces.

https://developers.google.com/cast/docs/reference/caf_receiver/cast.framework.breaks

Cheers